src/transform-sdk/cpp/src/transform_sdk.cc

```diff
@@ -424,6 +420,10 @@ bytes::value_type bytes_view::operator[](size_t n) const { return _start[n]; }
 bool bytes_view::operator==(const bytes_view& other) const {
     return std::ranges::equal(*this, other);
 }
+bytes_view::operator std::string_view() const {
```

Comment:
nit: looks like you're using this `explicit`...should it be declared so?

Reply:
No explicit doesn't take place in overlod resolution and it's only allowed at declaration

Reply:
Oh, it is declared explicit. My mistake. Thought I was looking at the declaration right here 👍